bin/commands/aux-create.js

```diff
@@ -1,26 +1,25 @@
-#! /usr/bin/env node
+#!/usr/bin/env node
+import dotenv from 'dotenv';
```

Comment:
Do we need the semicolons? I know I prefer not having them :) Isn't Prettier in here yet?

Reply:
No prettier yet, I'll remove all the semicolons